_sass/_layout.scss

```diff
@@ -295,6 +295,24 @@ pre code.hljs {
 		.disqus {
 			@include span-columns(12);
 		}
+		.post-navigation {
+			font-size: 14px;
```

Comment:
Let's stick to em's for font-sizing, that way everything scales if you override some Bourbon defaults. 

I believe the default it 16px, so 0.9em would be close to the 14px you're going for. I use 0.9em for other "smaller" text throughout the theme (i.e. site footer text)